components/capi/ctest/Makefile

```diff
@@ -0,0 +1,25 @@
+# This file is part of ICU4X. For terms of use, please see the file
```

Comment:
If we're eventually going to have a lot of examples for the C API, I'm wondering if using make is going to cause us unpleasant experiences in the future. I realize it is generic and widely available, but it isn't that nice to work with. On the other hand, if we choose something else, then it is an extra dependency, and something we need to get working in CI. Offhand, I was thinking of something like meson as an alternative build tool.

Reply:
I don't like make very much either, but it _is_ the de-facto C build system so I figured we should use it.

Reply:
I think we should use CMake so that the MSVC build files are auto-generated for Windows.

Reply:
If not using Make is an option we're willing to consider, we should probably have a discussion on C build systems before we chose one off hand. I've used CMake in the past and while it is definitely nicer to work with than Make, it still has some oddities. Other tools, like Meson, can also generate msvc build files, so it would be nice to determine our requirements and see what is out there that would be usable. I'd like to see us pick something as user friendly as possible for example code.

Reply:
That would be great; but I don't think we necessarily need to block on this just yet -- this is just a quick and dirty test for the API.

Reply:
For sure, don't block on this.